src/meta/src/manager/diagnose.rs

```diff
@@ -626,7 +626,7 @@ impl DiagnoseCommand {
         let compute_clients = ComputeClientPool::adhoc();
         for worker_node in &worker_nodes {
             if let Ok(client) = compute_clients.get(worker_node).await
-                && let Ok(result) = client.stack_trace().await
+                && let Ok(result) = client.stack_trace(StackTraceRequest::default()).await
```

Comment:
I'm thinking maybe json can be better for diagnose dump 🤔

Reason: we have periodical diagnose dump on cloud. So we could use that json to analyze more easily.

But the diagnose report is also human readable, so I'm a little hesitant.

Reply:
I'll add an API to convert JSON to human-readable text. This is mainly to assist with manual review in cases where bottleneck detect produces inaccurate analysis.

Reply:
>I'm thinking maybe json can be better for diagnose dump 

Update: now it's json.